src/org/zaproxy/zap/extension/replacer/ExtensionReplacer.java

```diff
@@ -96,7 +96,7 @@ private OptionsReplacerPanel getOptionsReplacerPanel() {
         return optionsReplacerPanel;
     }
 
-    protected ReplacerParam getParams() {
+    public ReplacerParam getParams() {

```

Comment:
Not sure exposing it is the best option, the rules are not immutable and not all of the changes done to them will be persisted, if we keep this change it's better to document that.

Reply:
What do you suggest instead?